### PR TITLE
data v2.1.0 changes

### PR DESCRIPTION
**Pull request by mDuo13**
Four new methods (documentation mostly done by the Data team; I've just polished it up slightly):
- Get Top Currencies
- Get Top Markets
- Get Account Transaction Stats
- Get Account Value Stats

Various whitespace changes (this happened automatically because I switched text editors)

`convertedAmount` -> `converted_amount`

Found a few cases of "Ripple, Inc" and replaced them accordingly

Fixed one or two cases of invalid JSON

### CODE REVIEW

File: content/reference-data-api.md [hunk 1]
```diff
@@ -1752,6 +1757,207 @@ Response:
 
 
 
+## Get Top Currencies ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/network/topCurrencies.js "Source")
+
+Returns the top currencies on the Ripple Consensus Ledger over a rolling 30 day window. The currencies are ordered from highest ranked to lowest, where the rank is determined by the volume and count of transactions and the number of unique counterparties. By default, returns the top currencies for the most recent date available. You can query historical data by date. _(New in [v2.1.0][])_
```
Comment (ryangyoung) [inline content/reference-data-api.md]: A "rolling 30 day window" is a little ambiguous. What exactly does it mean here? That it will always use the last 30 days of data? Should clarify that. 

Also, perhaps rephrase the second sentence so that the definition of the ranking comes first. Depending on how you rephrase the first sentence, you might be able to wedge the sort order description there (i.e., "Returns the top currencies on the RCL for the last 30 days, sorted from highest to lowest ranking. The rank is determined by...").

Finally, maybe rephrase the last sentence to something like "You can also query the top currencies for a specified date range using historical data" to make it's connection to the rest of the summary a little clearer.

File: content/reference-data-api.md [hunk 2]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+[Try it! >](data-api-v2-tool.html#get-top-currencies)
+
+
+#### Response Format ####
+
+A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| result | `success` | Indicates that the body represents a successful response. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Shouldn't the "Value" be "String" here? Seems odd to specify an actual value here.

File: content/reference-data-api.md [hunk 3]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+| date   | String - [Timestamp][] | The time at which this data was measured. |
```
Comment (ryangyoung) [inline content/reference-data-api.md:1795]: "Measured" doesn't seem like quite the right word here. Instead, maybe: "Collected," "obtained," or "calculated"?

File: content/reference-data-api.md [hunk 4]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+| count  | Integer | Number of results returned. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: What does a "result" mean here? The number of "Top Currency Objects"?

File: content/reference-data-api.md [hunk 5]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+| currencies | Array of Top Currency Objects | A currency + issuer, and the metrics measured for inclusion and ranking |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Shouldn't the "Value" (Type?) field define just the type? And any further description would be in the "Description" field (i.e., "Value: Array"; "Description: Array of Top Currency objects")? I could see that could be a little redundant, but that's what makes sense to me. Maybe there's a good reason not to do that.

Comment (ryangyoung) [inline content/reference-data-api.md]: The description could probably be more expansive, e.g.: "Array of Top Currency objects. Each Top currency object includes `currency` and `issuer` fields, and additional metrics used for inclusion and ranking.  See the Top Currency Object table for a detailed description of each field."

File: content/reference-data-api.md [hunk 6]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+
+Each Top Currency Object has the following fields:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| currency | String - [Currency Code][] | The currency this object describes |
```
Comment (ryangyoung) [inline content/reference-data-api.md:1803]: Would be great to have a standard include for common fields like this. As is, maybe mention the standard "in ISO 4217 format." And maybe link to the following definition, though that might not be necessary: https://ripple.com/build/rippled-apis/#specifying-currency-amounts (or https://ripple.com/build/rippled-apis/#currency-codes)

Never mind: I see the "Value" already links there.

File: content/reference-data-api.md [hunk 7]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+| issuer | String - [Address][] | The Ripple address that issues this currency |
```
Comment (ryangyoung) [inline content/reference-data-api.md:1804]: Maybe reuse the description from the link above, i.e.: "Unique account address of the entity issuing the currency. In other words, the person or business where the currency can be redeemed."

File: content/reference-data-api.md [hunk 8]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+*Most Recent*
+
+```
+GET /v2/network/top_currencies
+```
+
+*By Date*
+
+```
+GET /v2/network/top_currencies/2016-01-01
```
Comment (ryangyoung) [inline content/reference-data-api.md:1779]: Maybe, per the [API Template](https://ripplelabs.atlassian.net/wiki/display/DOCS/REST+API+Method+Template), add a note about this method not taking any URL parameters.

File: content/reference-data-api.md [hunk 9]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+| count  | Integer | Number of results returned. |
+| currencies | Array of Top Currency Objects | A currency + issuer, and the metrics measured for inclusion and ranking |
+
+Each Top Currency Object has the following fields:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| currency | String - [Currency Code][] | The currency this object describes |
+| issuer | String - [Address][] | The Ripple address that issues this currency |
+| avg_exchange_count | [String - Number][] | Daily average number of exchanges |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Link to Exchange Objects section: http://mduo13.github.io/ripple-dev-portal/reference-data-api.html#exchange-objects

File: content/reference-data-api.md [hunk 10]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+| avg_exchange_volume | [String - Number][] | Daily average volume of exchanges, normalized to XRP |
+| avg_payment_count | [String - Number][] | Daily average number of payments |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Link to Payment Objects section: http://mduo13.github.io/ripple-dev-portal/reference-data-api.html#payment-objects

File: content/reference-data-api.md [hunk 11]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+  ]
+}
+```
+
+
+
+## Get Top Markets ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/network/topMarkets.js "Source")
+
+Returns the top exchange markets on the Ripple Consensus Ledger over a rolling 30 day window.  The markets are ordered from highest ranked to lowest, where the rank is determined by the number and volume of exchanges and the number of counterparties participating.  By default, returns top markets for the latest date available.  You can query historical data by date. _(New in [v2.1.0][])_
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Rephrase according to suggestions for "Get Top Currencies."

File: content/reference-data-api.md [hunk 12]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+*Most Recent*
+
+```
+GET /v2/network/top_markets
+```
+
+*By Date*
+
+```
+GET /v2/network/top_markets/2016-01-01
```
Comment (ryangyoung) [inline content/reference-data-api.md:1879]: Add a URL Parameters section, similar to Get Top Currencies.

File: content/reference-data-api.md [hunk 13]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+[Try it! >](data-api-v2-tool.html#get-top-markets)
+
+
+#### Response Format ####
+
+A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| result | `success` | Indicates that the body represents a successful response. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Same comments as for the Get Top Currencies response fields:
- Actual field name shouldn't be in Value field.
- "Measured" doesn't seem quite right for the `date` field. 
- `Value` field for `markets` should probably only define type, with description going into `Description` field. 
- `Description` field for `markets` should be rephrased, similar to suggestion for Top Currencies object.

File: content/reference-data-api.md [hunk 14]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| base_currency | String - [Currency Code][] | The base currency for this market |
+| base_issuer | String - [Address][] | (Omitted if `base_currency` is XRP) The Ripple address that issues the base currency |
+| counter_currency | String - [Currency Code][] | The counter currency for this market |
+| counter_issuer | String - [Address][] | (Omitted if `counter_currency` is XRP) The Ripple address that issues the counter currency |
+| avg_base_volume | String | Daily average volume in terms of the base currency |
+| avg_counter_volume | String | Daily average volume in terms of the counter currency |
+| avg_exchange_count | String | Daily average number of exchanges |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Link to Exchanges Objects section.

File: content/reference-data-api.md [hunk 15]
```diff
@@ -3006,6 +3212,190 @@ Response:
 
 
 
+## Get Account Transaction Stats ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/accountStats.js "Source")
+
+Retrieve daily summaries of transaction activity for an account. _(New in [v2.1.0][].)_
```
Comment (ryangyoung) [inline content/reference-data-api.md:3220]: Sounds like the default is a daily summary for today's date, but you can optionally get summaries for other time ranges. Perhaps revise the description to reflect that.

File: content/reference-data-api.md [hunk 16]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+
+<!--</div>-->
+
+[Try it! >](data-api-v2-tool.html#get-account-transaction-stats)
+
+This method requires the following URL parameters:
+
+| Field    | Value  | Description |
+|----------|--------|-------------|
+| :address | String | Ripple address to query |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: _Minor and superficial:_ It looks like all the existing open-source reference docs have "Fields" in regular font. For internal docs, I used backticks around the field names because I thought the monospace font looked better in the PDF. Probably better to maintain consistency at this point than bother changing what's there already, but I'd be interested in your take for future docs, style guide, etc.

Comment (ryangyoung) [inline content/reference-data-api.md]: Another minor point: some descriptions end with periods, some don't. I think being consistent by adding periods even to sentence fragments is probably the way to go.

File: content/reference-data-api.md [hunk 17]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+
+
+Optionally, you can also include the following query parameters:
+
+| Field      | Value   | Description |
+|------------|---------|-------------|
+| start      | String  | UTC start time of query range. Defaults to start of current date. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Maybe: "Start time of query range, in UTC". 

Also: "Defaults to current date", unless the description for `end` is changed to "Defaults to end of current date." Whichever is more technically accurate.

File: content/reference-data-api.md [hunk 18]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| end        | String  | UTC end time of query range. Defaults to current date. |
+| limit      | Integer | Max results per page (defaults to 200). Cannot be more than 1000. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: "Maximum results per page." Default should probably be freed from parentheses.

File: content/reference-data-api.md [hunk 19]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| descending | Boolean | If true, sort results with most recent first. By default, sort results with oldest first. |
```
Comment (ryangyoung) [inline content/reference-data-api.md:3248]: Suggest: "If true, .... If false, sort results with oldest first. Defaults to `false.`"

File: content/reference-data-api.md [hunk 20]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| marker     | String  | [Pagination](#pagination) key from previously returned response. |
+| format     | String  | Format of returned results: `csv`,`json` defaults to `json` |
+
+
+#### Response Format ####
+A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| result | `success` | Indicates that the body represents a successful response. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: See comments on this for Get Top Currencies and Get Top Markets.

File: content/reference-data-api.md [hunk 21]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| count | Integer | Number of reports returned. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: "Number of Report Objects returned." And link to Report Objects section.

File: content/reference-data-api.md [hunk 22]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| rows | Array of Transaction Stats Objects | Daily summaries of account transaction activity for the given account. |
```
Comment (ryangyoung) [inline content/reference-data-api.md:3260]: See comments on this for Get Top Currencies and Get Top Markets.

File: content/reference-data-api.md [hunk 23]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+
+Each Transaction Stats Object has the following fields:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| date   | String - [Timestamp][] | The date this transaction stats object describes. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: The object doesn't describe the date exactly, but rather the transactions that occurred on that date, right? Suggest rephrasing to reflect that (if it is correct).

File: content/reference-data-api.md [hunk 24]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+  ]
+}
+```
+
+
+
+## Get Account Value Stats ##
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routesV2/accountStats.js "Source")
+
+Retrieve daily summaries of transaction activity for an account. _(New in [v2.1.0][].)_
```
Comment (ryangyoung) [inline content/reference-data-api.md:3318]: As with Get Account Transaction Stats, suggest revising this to reflect the fact that it is by default a daily summary, but can also be a summary of other time ranges.

File: content/reference-data-api.md [hunk 25]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| Field    | Value  | Description |
+|----------|--------|-------------|
+| :address | String | Ripple address to query |
+
+
+Optionally, you can also include the following query parameters:
+
+| Field      | Value   | Description |
+|------------|---------|-------------|
+| start      | String - [Timestamp][] | Start time of query range. Defaults to the start of the most recent interval. |
```
Comment (ryangyoung) [inline content/reference-data-api.md:3343]: In UTC? Suggest revising this and `end` in-line with description for Get Account TX Stats.

File: content/reference-data-api.md [hunk 26]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| end        | String - [Timestamp][] | End time of query range. Defaults to the end of the most recent interval. |
+| limit      | Integer | Max results per page (defaults to 200). Cannot be more than 1000. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Default is probably too important to be closed off in parens.

File: content/reference-data-api.md [hunk 27]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| marker     | String  | [Pagination](#pagination) key from previously returned response. |
+| descending | Boolean | If true, sort results with most recent first. By default, sort results with oldest first. |
```
Comment (ryangyoung) [inline content/reference-data-api.md:3347]: Suggest: "If true, .... If false, sort results with oldest first. Defaults to false."

File: content/reference-data-api.md [hunk 28]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| format     | String  | Format of returned results: `csv` or `json`. Defaults to `json`. |
+
+
+#### Response Format ####
+A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| result | `success` | Indicates that the body represents a successful response. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: See comments about this field for other methods.

File: content/reference-data-api.md [hunk 29]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| count | Integer | Number of reports returned. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: "Report Objects" and link to that section.

File: content/reference-data-api.md [hunk 30]
```diff
@@ -3006,6 +3212,190 @@ Response:
...
+| rows | Array of Value Stats Objects | Daily summaries of account value for the given account. |
+
+Each Value Stats Object has the following fields:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| date   | String - [Timestamp][] | This date this object describes. |
```
Comment (ryangyoung) [inline content/reference-data-api.md]: Same as Get Account TX Stats: The object doesn't describe the date exactly, but rather the value of that account on the specified date, right? Suggest rephrasing to reflect that (if it is correct).

File: content/reference-data-api.md (hunk 1 above)
Comment (mDuo13) [inline content/reference-data-api.md]: I don't want to lose the term "rolling window" because that is a well-defined statistics term, which makes it clear that multiple sub-samples overlap. That said, I rephrased as follows:

> Returns the top currencies on the Ripple Consensus Ledger, ordered from highest rank to lowest. The ranking is determined by the volume and count of transactions and the number of unique counterparties. By default, returns results for the 30-day rolling window ending on the current date. You can specify a date to get results for the 30-day window ending on that date.

File: content/reference-data-api.md (hunk 8 above)
Comment (mDuo13) [inline content/reference-data-api.md:1779]: Done.

File: content/reference-data-api.md (hunk 3 above)
Comment (mDuo13) [inline content/reference-data-api.md:1795]: Rephrased to be more descriptive:

> The end of the rolling window over which this data was calculated.

File: content/reference-data-api.md (hunk 4 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Well, the number of such objects in _this_ response. (This description is recycled from elsewhere in the API doc, where the `count` in one response may be less than the total number of results over a paginated set of responses.)

Changed to:

> Number of results in the `markets` field.

(and the appropriate variation for `currencies`)

File: content/reference-data-api.md (hunk 1 above)
Comment (ryangyoung) [inline content/reference-data-api.md]: Much better!

File: content/reference-data-api.md (hunk 6 above)
Comment (mDuo13) [inline content/reference-data-api.md:1803]: For the record, the standard include already exists. See lines 3605-3608 of the full file.

File: content/reference-data-api.md (hunk 5 above)
Comment (mDuo13) [inline content/reference-data-api.md]: For statically-typed programming languages, it's standard for the type of object contained in an array to be part of the data-type of that array, because arrays (typically) are limited to holding only one type of variable. Although JSON documents are not required to follow those rules, by putting the type of objects as part of the array definition, we can indicate clearly that users can make assume that the contents are all the same object type if they parse the API response using a statically-typed language.

I don't like saying, "See the Top Currency Object table" when the table is not explicitly named, but I did revise the description:

> The currency code, issuer, and metrics used for ranking this currency.

File: content/reference-data-api.md (hunk 7 above)
Comment (mDuo13) [inline content/reference-data-api.md:1804]: I'd rather keep the description shorter in the table. They can click the "Address" link to get the detailed version.

File: content/reference-data-api.md (hunk 9 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Done.

File: content/reference-data-api.md (hunk 10 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Done.

File: content/reference-data-api.md (hunk 5 above)
Comment (mDuo13) [inline content/reference-data-api.md]: On second thought, the description I wrote is misleading because it describes one member, not the field itself. Here's a better Description for the `currencies` field:

> The top currencies for this data sample. Each member represents one currency, by currency code and issuer.

File: content/reference-data-api.md [hunk 31]
```diff
@@ -1752,6 +1757,207 @@ Response:
...
+#### Response Format ####
+
+A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| result | `success` | Indicates that the body represents a successful response. |
+| date   | String - [Timestamp][] | The time at which this data was measured. |
+| count  | Integer | Number of results returned. |
+| markets | Array of Top Market Objects | A currency pair and the metrics measured for inclusion and ranking |
```
Comment (mDuo13) [inline content/reference-data-api.md]: Changed the "Description" as per Top Currencies above:

> The top markets for this data sample. Each member represents a currency pair.

File: content/reference-data-api.md (hunk 15 above)
Comment (mDuo13) [inline content/reference-data-api.md:3220]: Not quite. The default is an array of daily summaries covering the entire active history of the account. The query parameters can filter it down to fewer total days, results, or reverse order, but it pretty much always returns "daily summaries of transaction activity for an account."

File: content/reference-data-api.md (hunk 16 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Yes, I had noticed that about the "Fields" column. I chose non-monospace because I think it looks better and it's redundant to put everything in monospace when you know from context that they're field names. That said, I'm not wedded to one convention.

I'm with you on adding periods to the fragments for consistency. It's hard to catch them all though.

File: content/reference-data-api.md (hunk 17 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Rephrased this to match the better wording elsewhere in the doc.

File: content/reference-data-api.md (hunk 18 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Changed this and ~20 other cases to free the default from parentheses and replace "Max results" with "Maximum results"

File: content/reference-data-api.md (hunk 19 above)
Comment (mDuo13) [inline content/reference-data-api.md:3248]: I _think_ that should be accurate, but I'm hesitant to make this change across many API calls because there may be some weird cases where the API checks for the presence of the parameter rather than its value. (I don't _think_ the Data API has any of that, but there's at least one case in the `rippled` APIs, so I'm wary...)

I'm going to leave this as is for consistency with the rest of the document, rather than investigating every API method just to be sure.

File: content/reference-data-api.md (hunk 21 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Yeah, these aren't Report Objects. Changed the description to:

> Number of transaction stats objects in the `rows` field.

File: content/reference-data-api.md (hunk 22 above)
Comment (mDuo13) [inline content/reference-data-api.md:3260]: Leaving as-is, per my response there.

File: content/reference-data-api.md (hunk 23 above)
Comment (mDuo13) [inline content/reference-data-api.md]: I think it's debatable. Yes, in the broad sense it doesn't describe the date. I think it does describe the date "in terms of transaction stats." And it's clearly plainer language to phrase it this way than something like, "The date to which this Transaction Stats Object pertains."

I tried rephrasing this, but the best I could come up with is this:

> This object describes activity on this date.

File: content/reference-data-api.md (hunk 24 above)
Comment (mDuo13) [inline content/reference-data-api.md:3318]: As with the other, it's an array of daily summary objects, so no change here.

File: content/reference-data-api.md (hunk 25 above)
Comment (mDuo13) [inline content/reference-data-api.md:3343]: This description matches the convention used throughout the doc. The **Timestamp** link mentions the requirements (e.g. whether it has to be in UTC or not, which I think has changed over the course of the API's evolution)

File: content/reference-data-api.md (hunk 27 above)
Comment (mDuo13) [inline content/reference-data-api.md:3347]: As before, leaving this as-is.

File: content/reference-data-api.md (hunk 30 above)
Comment (mDuo13) [inline content/reference-data-api.md]: Changed to:

> This object describes activity on this date.

Not totally happy with that, but I guess it's better than the original wording.

File: content/reference-data-api.md (hunk 4 above)
Comment (ryangyoung) [inline content/reference-data-api.md]: Sounds good on both counts.

File: content/reference-data-api.md (hunk 6 above)
Comment (ryangyoung) [inline content/reference-data-api.md:1803]: Awesome! Didn't see the include until after I wrote my comment.

File: content/reference-data-api.md (hunk 7 above)
Comment (ryangyoung) [inline content/reference-data-api.md:1804]: Yep, that sounds good.

File: content/reference-data-api.md (hunk 15 above)
Comment (ryangyoung) [inline content/reference-data-api.md:3220]: Interesting, not what I would've expected. But if that's the case, then disregard my comment.

File: content/reference-data-api.md (hunk 16 above)
Comment (ryangyoung) [inline content/reference-data-api.md]: Cool. It might just be my eyes, but using monospace, bold, etc, seems more important for PDFs than HTML. Something to think about for the future.

File: content/reference-data-api.md (hunk 19 above)
Comment (ryangyoung) [inline content/reference-data-api.md:3248]: Oh man, I didn't think about a case where it just checks whether the field is there. In that case, makes sense to leave as is.

File: content/reference-data-api.md (hunk 23 above)
Comment (ryangyoung) [inline content/reference-data-api.md]: The rephrase is more what I was thinking of. I see your point, though, that, for the purposes of this call, the date is described by the transaction stats it returns. I'd vote for the rephrase, but it's your call.
